source/serialization/coresystem.rst

```diff
@@ -15,8 +15,11 @@ Voting Key Link
 VotingKeyLinkTransaction
 ========================
 
-Announce a VotingKeyLinkTransaction to associate a |BLS| public key with an account.
-Required for node operators willing to vote finalized blocks.
+This transaction is required for node operators willing to vote finalized blocks.
```

Comment:
This transaction is required for node operators wanting to vote for finalized blocks.